### PR TITLE
Aqua setup

### PR DESCRIPTION
**Pull request by jgiovaresco**
## Issue

https://gravitee.atlassian.net/browse/APIM-6303

## Description

Configure Aqua as a replacement for Snyk.
Aqua provides 3 features:
- Filesystem scans to check for vulnerabilities, sensitive data, sast, IaC/Pipeline misconfiguration. Result can be found at https://eu-1.cloud.aquasec.com/ah/#/supplychain/codeRepositories/repositories/784ae3af-98f7-4ef1-91b8-531aef3fc61e/builds/14930c7d-c6a5-4bca-915a-b3a5119c5a48/vulnerabilities
- Docker image scanning. Result can be found at https://eu-1.cloud.aquasec.com/ah/#/csp/images/CiCdScans
- Artifact registration to generate SBOM. Result can be found at https://eu-1.cloud.aquasec.com/ah/#/supplychain/releaseartifacts

Behavior:
- The filesystem scan is run all the time (for each PR, branch...)
- The Docker image scanning and Artifact registration is only done for `master`, support branches (4.x.x) and released version (4.4.1)

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-lqyilncdjb.chromatic.com)

### CODE REVIEW

File: .circleci/ci/src/workflows/workflow-pull-requests.ts [hunk 1]
```diff
@@ -49,6 +49,9 @@ import {
   PortalWebuiBuildJob,
   WebuiLintTestJob,
 } from '../jobs';
+import { aquasec } from '../orbs/aquasec';
+import { keeper } from '../orbs/keeper';
```
Comment (phiz71) [inline .circleci/ci/src/workflows/workflow-pull-requests.ts]: don't need if you import orbs on the next line

File: .circleci/ci/src/pipelines/tests/resources/build-rpm-and-docker-images/build-rpm-and-docker-images-prerelease-dry-run.yml [hunk 2]
```diff
@@ -74,10 +74,54 @@ jobs:
...
-            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet  -t graviteeio/apim-portal-ui:4.1.0-alpha.1 -t graviteeio/apim-portal-ui:4.1.0-alpha  -f ./gravitee-apim-portal-webui/docker/Dockerfile-from-download ./gravitee-apim-portal-webui/docker
+            docker buildx build --platform=linux/arm64,linux/amd64 --build-arg GRAVITEEIO_VERSION=4.1.0-alpha.1 --build-arg GRAVITEEIO_DOWNLOAD_URL=https://gravitee-dry-releases-downloads.cellar-c2.services.clever-cloud.com/pre-releases/graviteeio-apim/distributions --quiet -t graviteeio/apim-gateway:4.1.0-alpha.1 -t graviteeio/apim-gateway:4.1.0-alpha  -f ./gravitee-apim-gateway/docker/Dockerfile-from-download ./gravitee-apim-gateway/docker
```
Comment (ytvnr) [inline .circleci/ci/src/pipelines/tests/resources/build-rpm-and-docker-images/build-rpm-and-docker-images-prerelease-dry-run.yml:77]: looks a bit weird to me to have the version 4.1.0-alpha but it is not related to your PR directly :)

**Review by phiz71 — APPROVED** [review]
LGTM.
Just a minor thing to check before merging

**Review by ytvnr — APPROVED** [review]

**Review by ThibaudAV — APPROVED** [review]

File: .circleci/ci/src/pipelines/tests/resources/build-rpm-and-docker-images/build-rpm-and-docker-images-prerelease-dry-run.yml (hunk 2 above)
Comment (jgiovaresco) [reply .circleci/ci/src/pipelines/tests/resources/build-rpm-and-docker-images/build-rpm-and-docker-images-prerelease-dry-run.yml:77]: I'm not sure to understand, it is for when we release "pre-release" version of the product 🙈

**Pull request merged**